settings.py

```diff
@@ -1586,3 +1586,8 @@
     pass
 else:
     initialize(DATADOG_API_KEY, DATADOG_APP_KEY)
+
+
+CUSTOM_DASHBOARD_PAGE_URL_NAMES = {
+    'ews-ghana': 'dashboard_page'
+}
```

Comment:
can you move this closer to the other custom settings? e.g. maybe near `CUSTOM_MODULES`